pages/index.jsx

```diff
@@ -302,6 +265,18 @@ const MobileMenuLinks = styled.ul`
 const IssuesPage = () => {
   const [modalOn, setModalOn] = useState(false);
   const [mobileMenuOn, setMobileMenuOn] = useState(false);
+  const { isLoading, error, data } = useQuery(["myData"], () =>
+    fetch("https://prolog-api.profy.dev/content-page/home").then((res) =>
+      res.json()
+    )
+  );
```

Comment:
Great job fetching the data here. Have a look at the useGetIssues hook to see how to create a custom hook for this. Also follow the fetch function there to see how you can add the API request to the data layer. Mixing UI code and data fetching code like this will end up in a mess once the application grows